src/rust/Cargo.lock

```diff
@@ -1965,12 +1965,12 @@ dependencies = [
 name = "grapl-tracing"
 version = "0.0.1"
 dependencies = [
- "opentelemetry 0.17.0",
```

Comment:
Why are Rust dependencies being updated in this PR?

Reply:
Removed